odxtools/audience.py

```diff
@@ -24,10 +27,10 @@ def enabled_audiences(self):
     def disabled_audiences(self):
         return self._disabled_audiences
 
-    def _resolve_references(self, id_lookup):
-        self._enabled_audiences = [id_lookup[ref]
+    def _resolve_references(self, odxlinks):
```

Comment:
Some type hinting would be nice
```suggestion
    def _resolve_references(self, odxlinks: OdxLinkDatabase):
```